### PR TITLE
chore(features) Remove new-issue-ui feature checks.

### PR DESCRIPTION
**Pull request by markstory**
Remove the new-issue-ui feature registration and all places it was checked. This feature has been mainlined for a while.

Refs APP-729

### CODE REVIEW

File: src/sentry/static/sentry/app/components/group/sidebar.jsx [hunk 1]
```diff
@@ -259,24 +256,25 @@ const GroupSidebar = createReactClass({
...
-              topValues={topValues}
-              name={tag.name}
-              data-test-id="group-tag"
-              orgId={orgId}
-              projectId={projectId}
-              group={group}
-            />
-          );
-        })}
+        {this.state.tagsWithTopValues &&
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/group/sidebar.jsx]: Prettier felt strongly about this. 🤷‍♂️

File: src/sentry/static/sentry/app/views/groupDetails/actions.jsx [hunk 2]
```diff
@@ -285,37 +284,6 @@ const GroupDetailsActions = createReactClass({
...
-                </a>
-              </div>
-            );
-          })
-        )}
-        {group.pluginIssues &&
-          !orgFeatures.has('new-issue-ui') &&
-          group.pluginIssues.map(plugin => {
-            return <IssuePluginActions key={plugin.slug} plugin={plugin} />;
-          })}
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/views/groupDetails/actions.jsx]: Are we definitely able to completely remove these plugin buttons?

Comment (macqueen) [reply src/sentry/static/sentry/app/views/groupDetails/actions.jsx]: i'm pretty sure we can, given `new-issue-ui` was i think on for everyone and these all have a `!orgFeatures.has('new-issue-ui')` check

but @MeredithAnya can you confirm?

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/groupDetails/actions.jsx]: Ahh, you're right about the `orgFeatures.has('new-issue-ui')`. lgtm

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**